two-data-centers-in-one-city-deployment.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Two Data Centers in One City Deployment
```

Comment:
I think it replaces the doc https://github.com/pingcap/docs/blob/master/synchronous-replication.md. maybe remove it.

Reply:
Yes. Please help delete this file and add alias in two-data-centers-in-one-city-deployment.md. Thanks!

Reply:
fixed in https://github.com/pingcap/docs/pull/6182/commits/cefc861a04d974194ae016b9bf0f124165217697.